### ISSUE TITLE
train

### ISSUE THREAD

**Issue by daocaorenzs** [bearpaw/pose-attention#13]
Hi,
When I was training, the accuracy of the training set was very high, but the accuracy of the validation set was very low, about 30%. I trained a total of 100 epochs, increased the number of iterations in the source code, and correspondingly reduced the batchsize, but I can ensure that every picture has been trained, so who can tell me why? thx.

**Comment by 738654805** [comment]
@daocaorenzs i met the same problem.it is so nice of you to share if your could help me to solve this problem.
